src/Main.java

```diff
@@ -1,8 +1,45 @@
+import java.util.Scanner;
+
+
 public class Main {
 
-    public static void main (String[] args){
-        int[] numeric = { 1,2,3,4,5,6,7,8,9,0};
+    public static void main (String[] args) {
+        int[] numeric = {1, 2, 3, 4, 5, 6, 7, 8, 9, 0};
         String[] alfabetic = {"een", "twee", "drie", "vier", "vijf", "zes", "zeven", "acht", "negen"};
 
+        // ik ben er niet meer achter gekomen welke argumenten ik mee kon geven aan de parameters. Maar ik ben benieuwd of ik de rest van de code wel juist had. Ik kon het niet testen
```

Comment:
Je was er bijna:
Translator translator(numeric, alfabetic)

Letop! In de zelfde volgorde als de constructor